spec/memo_wise_spec.rb

```diff
@@ -483,6 +483,58 @@ def child_class_method
           expect(instance.child_class_method_counter).to eq(1)
         end
       end
+
+      context "when the class included a module which includes another module with `.included`" do
+        let(:module_to_be_included_in_another_module) do
+          Module.new do
+            prepend MemoWise
+
+            def self.included(base)
+              base.class_eval do
+                @internal_state = true
+              end
+            end
+
+            def method1
+              raise "internal error" unless self.class.internal_state
+
+              false
+            end
+            memo_wise :method1
+          end
+        end
+
+        let(:module_to_be_included_in_class) do
+          Module.new do
+            def self.included(base)
+              base.class_eval do
+                include ModuleToBeIncludedInAnotherModule
+              end
+            end
+          end
+        end
+
+        let(:klass) do
+          Class.new do
+            def self.internal_state
+              @internal_state ||= false
+            end
+            include ModuleToBeIncludedInClass
+          end
+        end
+
+        let(:instance) { Klass.new }
+
+        before(:each) do
+          stub_const("ModuleToBeIncludedInAnotherModule", module_to_be_included_in_another_module)
+          stub_const("ModuleToBeIncludedInClass", module_to_be_included_in_class)
+          stub_const("Klass", klass)
+        end
+
+        it "runs `.included` in modules" do
+          expect(instance.send(:method1)).to be true
```

Comment:
Mistake when simplifying spec
```suggestion
          expect(instance.send(:method1)).to be false
```